tests/async.rs

```diff
@@ -768,7 +757,7 @@ fn prepared_nulls() {
             assert!(!params[1].value.is_null());
             assert_eq!(
                 params[0].coltype,
-                myc::constants::ColumnType::MYSQL_TYPE_SHORT
+                myc::constants::ColumnType::MYSQL_TYPE_NULL
```

Comment:
Weird. It's wild that this _used_ to work.

Reply:
I found this surprising too, but it looks correct now and I didn't dig much into why it worked earlier.